Documentation/Metrics/arangodb_revision_tree_buffered_memory_usage.yaml

```diff
@@ -0,0 +1,24 @@
+name: arangodb_revision_tree_buffered_memory_usage
+introducedIn: "3.12.0"
+help: |
+  Total memory usage of buffered updates for all revision trees.
+unit: bytes
+type: gauge
+category: Replication
+complexity: simple
```

Comment:
Is this really simple, i.e. it should be visible on the most simple dashboard? I would go for `advanced`.